src/Common/src/TypeSystem/Common/TypeSystemContext.cs

```diff
@@ -66,6 +66,13 @@ public virtual ModuleDesc ResolveAssembly(AssemblyName name, bool throwIfNotFoun
             return null;
         }
 
+        public virtual ModuleDesc ResolveModule(AssemblyName name, bool throwIfNotFound = true)
```

Comment:
If you want to add support for multi-module assemblies, it should be done in separate PR.